frontend/src/Admin/containers/AssignJudgesTable.js

```diff
@@ -0,0 +1,84 @@
+import { graphql, compose } from 'react-apollo'
+import { connect } from 'react-redux'
+
+import {
+  feetchJudgesByAssignmentForShow,
```

Comment:
feetch or fetch?

Reply:
Good catch. Copied and pasted this from `actions.js` which is also spelled wrong, but since they're the same, it doesn't cause issues.